js/libs/keycloak-admin-client/package.json

```diff
@@ -40,7 +40,8 @@
     "camelize-ts": "^2.3.0",
     "lodash-es": "^4.17.21",
     "url-join": "^5.0.0",
-    "url-template": "^3.1.0"
+    "url-template": "^3.1.0",
+    "undici": "^5.21.0"
```

Comment:
I don't feel comfortable adding `undici` as a dependency. I am worried this will bloat the size of the bundle on the client-side (browsers), where this dependency does basically nothing.

Reply:
@jonkoops 
I would do all the needed stuff, if I know, that the change will be merged. 
Our usecase is running api tests with playwright against the keycloak. But if this library usage is only for frontend, than I understand that this functionality is not needed and I will have to patch the lib on my side.

Pls write me a short answer if my patch would be useful or it is not needed ;)